src/loaders/svg/tvgSvgUtil.cpp

```diff
@@ -290,7 +292,10 @@ string svgUtilBase64Decode(const char *src)
 {
     if (!src) return nullptr;
 
-    auto length = strlen(src);
+    const char* x = src;
```

Comment:
How about `auto` instead of const char*?

Reply:
sorry, I forgot about that... changed